src/test/java/core/basesyntax/FruitTransactionProcessorServiceTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitTransactionProcessorService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.FruitTransactionProcessorServiceImpl;
+import core.basesyntax.strategy.OperationHandlerStrategy;
+import core.basesyntax.strategy.OperationHandlerStrategyImpl;
+import core.basesyntax.strategy.handlers.BalanceHandler;
+import core.basesyntax.strategy.handlers.OperationHandler;
+import core.basesyntax.strategy.handlers.PurchaseHandler;
+import core.basesyntax.strategy.handlers.ReturnHandler;
+import core.basesyntax.strategy.handlers.SupplyHandler;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionProcessorServiceTest {
+    private static final Path TEST_FRUIT_CSV_FILE_PATH =
+            Path.of("src/test/resources/fruits_test.csv");
+    private static final Path TEST_REPORT_FILE_PATH =
+            Path.of("src/test/resources/report_test.csv");
+    private static List<String> fruitsOperationsList;
+    private static Map<String, Integer> expectedFruitsQuantityMap;
+    private static FruitTransactionProcessorService fruitTransactionProcessorService;
+
+    @BeforeClass
+    public static void initFruitTransactionProcessorService() {
+        FruitService fruitService = new FruitServiceImpl(new FruitDaoImpl());
+        Map<String, OperationHandler> operationsHandlerMap = new HashMap<>();
+        operationsHandlerMap.put("b", new BalanceHandler(fruitService));
+        operationsHandlerMap.put("s", new SupplyHandler(fruitService));
+        operationsHandlerMap.put("r", new ReturnHandler(fruitService));
+        operationsHandlerMap.put("p", new PurchaseHandler(fruitService));
+        OperationHandlerStrategy operationHandlerStrategy =
+                new OperationHandlerStrategyImpl(operationsHandlerMap);
+        fruitTransactionProcessorService =
+                new FruitTransactionProcessorServiceImpl(operationHandlerStrategy);
+    }
+
+    @BeforeClass
+    public static void initFruitsOperationsList() {
+        try {
+            fruitsOperationsList = Files.readAllLines(TEST_FRUIT_CSV_FILE_PATH);
+        } catch (IOException e) {
+            throw new RuntimeException("Could not read file", e);
+        }
+    }
+
+    @BeforeClass
```

Comment:
let's create one method with @BeforeClass and call it init() and call inside of it all these three methods